app/enterprise/1.3-x/kong-kubernetes/install.md

```diff
@@ -14,6 +14,9 @@ Before installing Kong for Kubernetes Enterprise, be sure you have the following
 - **Kubernetes cluster**: You can use [Minikube](https://kubernetes.io/docs/setup/minikube/) or a [GKE](https://cloud.google.com/kubernetes-engine/) cluster. Kong is compatible with all distributions of Kubernetes. 
 - **kubectl access**: You should have `kubectl` installed and configured to communicate to your Kubernetes cluster.
 - A valid Kong Enterprise License
+  * If you have a license, see https://docs.konghq.com/enterprise/latest/deployment/licensing/
+  * If you need a license, request a demo at https://konghq.com/request-demo/
+  * Or, try out Kong for Kubernetes Enterprise using a live tutorial at https://kubecon.konglabs.io/
```

Comment:
Do we know how long this tutorial will be available? If the tutorial site reaches EOL, the link to it might still work, hence it will pass the broken link test--but the user won't be able to do anything. 

If we know the EOL, we can remember to remove this link.

```suggestion
  * Or, try out Kong for Kubernetes Enterprise using a live tutorial at [https://kubecon.konglabs.io/](https://kubecon.konglabs.io/)
```